dev-python/pylama/pylama-7.4.3.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_{4,5,6}} )
+
+inherit distutils-r1
+
+DESCRIPTION="Code audit tool for Python and JavaScript."
+
+HOMEPAGE="https://github.com/klen/pylama"
+SRC_URI="https://github.com/klen/pylama/archive/${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="LGPL-3"
+
+SLOT="0"
+KEYWORDS="~amd64"
```

Comment:
~x86 keyword is missing.

Reply:
I didn't technically test it on x86, only on amd64 so I didn't think I should've put that keyword in. I updated the dependencies though. It should be good now!